org.idempiere.zk.breeze.theme/src/web/theme/default/css/fragment/desktop.css.dsp

```diff
@@ -16,16 +16,27 @@
 }
 
 .desktop-header {
-	background-color: #E4E4E4;
 	width: 100%;
 	height: 46px;
 	border-bottom: 1px solid #C5C5C5 !important;
 	padding-left: 4px;
 }
+.status-P {
+	background-color: #E4E4E4;
+}
+.status-E {
+	background-color: #90EE90;
+}
+.status-I {
+	background-color: #FA8072;
+}
```

Comment:
we either leave it as before this ticket or this should be in sync with iceblue_c

Reply:
> we either leave it as before this ticket or this should be in sync with iceblue_c

done